ios/FileUtils.m

```diff
@@ -108,36 +163,69 @@ @implementation FileUtils
  * @returns The height and width (x,y), of the video or image in pixels.
  */
 RCT_EXPORT_METHOD(
-                  getDimensions:(NSString *)path (NSString *)type
+                  getDimensions:(NSString *)path
+                  type: (NSString *)type
                   resolver: (RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject
                   )
 {
-    if (type == "image") {
-        UIImage *image = [UIImage imageWithContentsOfFile:path];
+    NSURL *referenceUrl = [NSURL URLWithString:path];
+    
+    if ([type isEqualToString:@"image"]) {
+        CGImageSourceRef sourceRef = CGImageSourceCreateWithURL((CFURLRef)referenceUrl, NULL);
         
-        if (image == nil) {
+        if (sourceRef == nil) {
             reject(
-                   @"QSRNFU-20",
+                   @"QSRNFU-30",
                    @"The method could not initialize the image from the specified file",
                    nil
                    );
             return;
         }
         
-        resolve(image.size.width, image.size.height)
+        CGFloat width = 0.0f, height = 0.0f;
+        CFDictionaryRef imageProperties = CGImageSourceCopyPropertiesAtIndex(sourceRef, 0, NULL);
+        CFRelease(sourceRef);
+        
+        CFNumberRef widthNum  = CFDictionaryGetValue(imageProperties, kCGImagePropertyPixelWidth);
+        if (widthNum != NULL) {
+            CFNumberGetValue(widthNum, kCFNumberCGFloatType, &width);
+        }
+
+        CFNumberRef heightNum = CFDictionaryGetValue(imageProperties, kCGImagePropertyPixelHeight);
+        if (heightNum != NULL) {
+            CFNumberGetValue(heightNum, kCFNumberCGFloatType, &height);
+        }
```

Comment:
We could extract this out to a helper method for extracting kCGImage properties? i.e.

```
CFNumberRef widthNum = [CFProperty value:imageProperties key:kCGImagePropertyPixelWidth]
CFNumberRef heightNum = [CFProperty value:imageProperties key:kCGImagePropertyPixelHeight]
```

For example ^

Reply:
No longer needed since we're getting image dimensions from RN Core lib now.